parts/linux/cloud-init/artifacts/components.json

```diff
@@ -60,24 +74,31 @@
       ]
     },
     {
-      "downloadURL": "mcr.microsoft.com/containernetworking/cni-dropgz:*",
+      "downloadURL": "mcr.microsoft.com/containernetworking/azure-ipam:*",
```

Comment:
any risk for this renaming? Anyone expecting the old name?

Reply:
this is an addition, it shows as a rename due to some quirk of git diff. this block has simply moved down: https://github.com/Azure/AgentBaker/pull/4256/files/aec8a803c11e8bd9b9fbc493eb27f212425245cd#diff-1af88d0f330c2b73c9a7dbd8c9ba3c649c0d192c40955f458e4050e62f0bbfd2R92
we still keep the old image name during this transition